src/tlo/methods/healthsystem.py

```diff
@@ -1472,8 +1472,8 @@ def enforce_priority_policy(self, hsi_event) -> int:
             return _priority_ranking
 
         else:  # If treatment is not ranked in the policy, issue a warning and assign priority=3 by default
-            warnings.warn(UserWarning(f"Couldn't find priority ranking for TREATMENT_ID \n"
-                                      f"{hsi_event.TREATMENT_ID}"))
+#           warnings.warn(UserWarning(f"Couldn't find priority ranking for TREATMENT_ID \n"
+#                                     f"{hsi_event.TREATMENT_ID}"))
```

Comment:
I think this should be uncommented

Reply:
it's because the priority ranking for all TREATMENT_ID must be declared in the PRIORITY_RANKING file

Reply:
Thanks @tbhallett   Is it enough to insert these in default tab or do I need to insert in each tab ?

Reply:
It needs to be on every tab -- as that is how you make the decision for the priority accorded to these treatments, under each regime (prioritise RMNCH, prioritise vertical programs etc.). I think you can probably just copy the patterns used for the other cancers.

Reply:
ok thanks Tim